Description: Reunify default BUFFER_SIZE on ARM64 to avoid crashes in DYNAMIC_ARCH
Origin: upstream, https://github.com/xianyi/OpenBLAS/pull/2963/commits/7f26be4802042d7c54bd1645c54adc3e2ff72d50
Bug: https://github.com/xianyi/OpenBLAS/issues/2959
Bug-Debian: https://bugs.debian.org/970558
Last-Update: 2020-11-01
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/common_arm64.h
+++ b/common_arm64.h
@@ -142,14 +142,8 @@ REALNAME:
 #define HUGE_PAGESIZE   ( 4 << 20)
 
 #ifndef BUFFERSIZE
-#if defined(CORTEXA57)
-#define BUFFER_SIZE     (20 << 20)
-#elif defined(TSV110) || defined(EMAG8180)
 #define BUFFER_SIZE     (32 << 20)
 #else
-#define BUFFER_SIZE     (16 << 20)
-#endif
-#else
 #define BUFFER_SIZE	(32 << BUFFERSIZE)
 #endif
 
